Pyvista tests breaking

Looks like latest pyvista 0.27.2 changed some internal APIs:

```python
    def pyvista_render_window():
        """
        Allow to download and create a more complex example easily
        """
        from pyvista import examples
        sphere = pv.Sphere() #test actor
        globe = examples.load_globe() #test texture
        head = examples.download_head() #test volume
        uniform = examples.load_uniform() #test structured grid
    
        scalars=sphere.points[:, 2]
>       sphere._add_point_array(scalars, 'test', set_active=True) #allow to test scalars
E       AttributeError: 'PolyData' object has no attribute '_add_point_array'
```

Cc @xavArtley

--- a/panel/tests/pane/test_vtk.py
+++ b/panel/tests/pane/test_vtk.py
@@ -68,7 +68,8 @@ def pyvista_render_window():
     uniform = examples.load_uniform() #test structured grid
 
     scalars=sphere.points[:, 2]
-    sphere._add_point_array(scalars, 'test', set_active=True) #allow to test scalars
+    sphere.point_arrays['test'] = scalars #allow to test scalars
+    sphere.set_active_scalars('test')
 
     uniform.set_active_scalars("Spatial Cell Data")
 

--- a/setup.py
+++ b/setup.py
@@ -137,7 +137,7 @@ extras_require = {
         'jupyter_bokeh',
         'django',
         'channels',
-        'pyvista <0.27', # temporary fix for tests 
+        'pyvista',
         'ipywidgets',
         'ipywidgets_bokeh',
         'ipyvolume',

